CONTRIBUTING.md

```diff
@@ -29,7 +29,23 @@ Feature requests are welcome, but take a moment to find out whether your idea fi
 
 ## Plugin requests
 
-Plugin submissions and requests are a great way to improve Streamlink. Requests should be as detailed as possible and dedicated to only a single streaming service. Information about the service as well as explicit URLs for live streams or recordings are encouraged so that the requested plugin can be easily implemented. Please be aware that plugins for streaming services that are using DRM protections will not be implemented.
+Plugin submissions and requests are a great way to improve Streamlink. Requests should be as detailed as possible and dedicated to only a single streaming service. Information about the service as well as explicit URLs for live streams are required, as are details regarding the website, what it is, who runs it, how it adds to Streamlink, etc. This allows for easier plugin review and implementation. We reserve the right to refuse to implement, develop, or update any plugin. In addition we may remove existing plugins at our own discretion.
+
+Plugins which fall under the following categories will not be implemented or considered and the request will be closed:
+
+1. Any kind of streaming service that uses DRM protection
+
+2. Sites which are hosting stolen content as their main source of content
+
+3. Sites which are primarily rehosting content that is available from a legitimate source (TV shows, sports, news, etc.)
+
+4. Sites which require any sort of cable login or subscription
+
+5. NSFW sites of a pornographic nature (cam sites, porn sites, etc.)
+
+6. Sites which are mostly hosting VODs (stolen or legitimate), or require changes to existing code to watch VODs
```

Comment:
> require changes to existing code to watch VODs

this opinion from me, was more meant to get a discussion,
if Streamlinks **main focus** is still about **Livestreams** or if it's already about **VODs and Livestreams**.

---

like I said before for me small changes are ok for VOD support,
but if the changes are completely different it's sometimes not worth it.

Reply:
Especially if youtube-dl already does a good job of it :)

Reply:
I'll modify it to note that it's large changes to existing code.